test/errors/managedRecord-detailed.good

```diff
@@ -0,0 +1,15 @@
+─── error in managedRecord.chpl:4 [MemManagementNonClass] ───
+  Cannot use memory management strategy 'shared' with record 'MyRecord'.
+      |
+    4 | var r2 = new shared MyRecord();
+      |                     ⎺⎺⎺⎺⎺⎺⎺⎺    
+      |
+  Memory management strategies can only be used with classes.
+  'MyRecord' declared as record here:
+      |
+    1 | record MyRecord {
+    2 |   
+    3 | }
+      |
+  Remove the 'shared' keyword to fix this error, or define 'MyRecord' as a class.
```

Comment:
I'm not sure how much these hints should be taken as gospel vs. potential paths to follow.  E.g., there may be many reasons that `MyRecord` cannot or should not be declared as a class, so I feel like this reads a bit naively like "All you need to do to fix this error is make this simple change" when that may not be appropriate (whereas "remove 'shared'" feels much more unambiguously reasonable to me).  Even a wiggle word like "or consider defining 'MyRecord' as a class" makes it feel much safer to me.